lib/registration.pm

```diff
@@ -509,5 +515,35 @@ sub scc_deregistration {
     }
 }
 
+# Some sle product addons name should be changed for sle15
+# 1, Some extensions / modules are changed since sle15:
+# Advanced Systems Management: packages are moved to SLE 15 base system
+# SLE-HA-GEO extension is merged to SLE-HA extension
+# SDK extension becomes Development-Tools module since sle15
+# Toolchain module: packages are moved to Development module
+# 2, Different addon names used between yast_scc_registration and
+# %SLE15_MODULES, such as:
+# lgm -- legacy
+# wsm -- script
+# Not good idea to use different module names, we have to bridge the
+# gap here to use existing needles
+sub rename_scc_addons {
+    return unless get_var('SCC_ADDONS') && sle_version_at_least('15');
```

Comment:
For testing SLE version you also need to use `is_sle` like this: `is_sle && sle_version_...`.

Reply:
Maybe we can just ignore the is_sle here? It is unlikely to define SCC_ADDONS openSUSE testing.

Reply:
Yes and no :-) Because `sle_version_at_least` can't just be use alone to test SLE OS. It's not just me (I had the same comments on my PR), it's just "needed".

I plan to try to modify this in the future, I need to open a discussion on poo# and/or #testing for this.

Reply:
Actually here we can leave without it. `sle_version_at_least` does not "need" `is_sle` it is rather that `sle_version_at_least` might not behave as one would expect on openSUSE. But for now openSUSE does not use `lib/registration.pm` at all so we should not need to care that much.